game-core/src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -191,12 +191,9 @@ protected void launchInNewThread(final Component parent) {
           });
           stopGame();
         } catch (final RuntimeException e) {
-          final String errorMessage = "Unrecognized error occurred: " + e.getMessage() + ", if this is a repeatable "
-              + "error please make a copy of this savegame and report to:\n" + UrlConstants.GITHUB_ISSUES;
+          final String errorMessage = "Unrecognized error occurred. If this is a repeatable error, "
+              + "please make a copy of this savegame and report to:\n" + UrlConstants.GITHUB_ISSUES;
```

Comment:
Removed a redundant copy of the error message.  The log formatter will include it in the final log message.